source/gwork/include/Gwork/ControlList.h

```diff
@@ -48,17 +49,17 @@ namespace Gwk
             list.remove(control);
         }
 
-        void Add(const List& list)
+        void Add(const List& _list)
```

Comment:
The local variable list shadows the class variable list; this causes clang to complain, and makes it kind of hard to reason about the code.